site/src/scaffold/scripts/form-submit.ts

```diff
@@ -0,0 +1,49 @@
+export function handleMockFormSubmit() {
+	document.querySelectorAll('form')?.forEach(form => {
+		form.addEventListener('submit', function (e) {
+			e.preventDefault();
```

Comment:
I think this might be interfering with the form behavior? Is using `action="#"` or `action="?"` as you've already done enough?

I think it might confuse folks to see the alert prior to a successful submission- perhaps listen for the form behavior's `beforesubmit` event instead?

Reply:
Added listener for `beforeSubmit` and replaced the alert with adding the formdata as text at the bottom of the form.